plugins/git/internal-functions

```diff
@@ -193,6 +193,20 @@ cmd-git-sync() {
 
   verify_app_name "$APP"
 
+  if [[ -d "$DOKKU_LIB_ROOT/data/git/$APP" ]]; then
+    if has_tty eq 0; then
+      cmd-git-unlock "$APP"
```

Comment:
We should _ask_ for confirmation before attempting to delete anything (thats the point of me asking you to check for a tty lol).

Reply:
Ok, I will try to make this work and same for tests.